src/main/scala/com/evolution/scache/Cache.scala

```diff
@@ -335,10 +780,35 @@ object Cache {
         }
     }
 
-    /**
-      * Does not run `value` concurrently for the same key
-      * release will be called upon key removal from the cache
-      * In case of none returned, value will be ignored by cache
+    /** Gets a value for specific key, or tries to load it.
+      *
+      * The difference between this method and [[Cache#getOrUpdate1]] is that
+      * this one allows the loading function to fail finding the value, i.e.
+      * return [[scala.None]].
+      *
+      * Note, that this may not come for free in some implementations, i.e. some
+      * implementations use exceptions to bypass the loading mechanism
+      * internally, so the performance may suffer in this case.
```

Comment:
> some
       * implementations use exceptions to bypass the loading mechanism
       * internally, so the performance may suffer in this case.


I think we should remove this, there is no performance penalty because of `NoStackTrace` exception

Reply:
Done